modules/ROOT/pages/ext-analytics-elk.adoc

```diff
@@ -0,0 +1,173 @@
+= Analyze Business and API Data using ELK
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use the Elastic Stack (ELK) to analyze the business data and API analytics generated by Anypoint Platform Private Cloud Edition (Anypoint Platform PCE). You can use Filebeat to process Anypoint Platform log files, insert them into an Elasticsearch database, and then analyze them with Kibana.
+
+The following image shows an example of how API data appears in Kibana:
+
+image::kibana-example.png[]
+
+== Prerequisites
+
+* Required software
```

Comment:
Suggest making this Level-1 list item parallel with its peers, either by making all three fragments or all three sentences.